tests/e2e/steps_partial_set_security.go

```diff
@@ -1391,11 +1391,11 @@ func stepsValidatorsPowerCappedChain() []Step {
 				ChainID("consu"): ChainState{
 					ValPowers: &map[ValidatorID]uint{
 						// the powers of the validators on the consumer chain are different from the provider chain
-						// because the consumer chain is power capped. Note that the total power is 600 (= 194 + 203 + 203)
-						// and 203 / 600.0 = 0.338 < 34% that is the power cap.
```

Comment:
We fixed a rounding issue based on "Inaccurate Rounding of maxPower" suggestion and hence those numbers had to be changed.